### PR TITLE
added index2.html A and button.css file

### PR DESCRIPTION
**Pull request by raulgutierrez224**
added dispatch button

### CODE REVIEW

File: src/app/components/homepage/ATLAS job list.code-workspace [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (looksliketyler) [inline src/app/components/homepage/ATLAS job list.code-workspace:1]: what is this file?

File: src/app/components/homepage/button.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+body{
```
Comment (looksliketyler) [inline src/app/components/homepage/button.css:1]: seems this file isnt in use

File: src/app/components/homepage/index2.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (looksliketyler) [inline src/app/components/homepage/index2.html:1]: this file isnt in use, probably remove it

File: src/app/components/homepage/workspace.code-workspace [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (looksliketyler) [inline src/app/components/homepage/workspace.code-workspace:1]: what is this file?

File: src/app/components/homepage/homepage.component.html [hunk 5]
```diff
@@ -46,21 +45,46 @@ <h4><b>Your Stuff (LMNO_USA_TECH)</b></h4>
...
             </div>
           </div>
         </div>
       </ng-container>
     </div>
   </div>
+  <div class="row">
+    <div class="col-md-12">
+      <button type="button" class="btn btn-danger">
+        {{ ReadMore ? "Dispatch" : "Dispatch" }}
```
Comment (looksliketyler) [inline src/app/components/homepage/homepage.component.html:86]: this ternary will always display `Dispatch`    theres no need for this ternary

File: src/app/components/homepage/homepage.component.ts [hunk 6]
```diff
@@ -8,6 +8,11 @@ import { WeatherService } from 'src/app/services/weather.service';
   styleUrls: ['./homepage.component.scss']
 })
 export class HomepageComponent implements OnInit {
+  
+  public ReadMore:boolean = false;
+  public onclick(): void {
```
Comment (looksliketyler) [inline src/app/components/homepage/homepage.component.ts:13]: this method isnt implemented anywhere

**Review by looksliketyler — CHANGES_REQUESTED** [review]
good initial work @raulgutierrez224.   this is pretty good for a first jump into Angular...you can make it better pal :)

File: src/app/components/homepage/ATLAS job list.code-workspace (hunk 1 above)
Comment (raulgutierrez224) [reply src/app/components/homepage/ATLAS job list.code-workspace:1]: I deleted it. it was a file I saved wrong

File: src/app/components/homepage/button.css (hunk 2 above)
Comment (raulgutierrez224) [reply src/app/components/homepage/button.css:1]: file deleted

**Comment by micah-wehrle** [comment]
no significant changes or changes incorporated via other branches. deleting ranch as pr closed.

**Pull request closed without merging**